contracts/test/e2e/AnyToAnySwapAndForward.t.sol

```diff
@@ -69,8 +69,14 @@ contract AnyToAnySwapAndForwardTest is TestHelper {
 
     // destination
     // set up destination swap params
-    bytes
-      memory oneInchApiDataUsdcToArb = hex"e449022e0000000000000000000000000000000000000000000000000000000005f5e100000000000000000000000000000000000000000000000004188a80f4c2e52ccf00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000001800000000000000000000000cda53b1f66614552f834ceef361a8d12a0b8dad8cfee7c08";
+    uint256 usdcAmount = 100000000;
+    uint256 minReturn = 75555343887264722127;
+    address USDC_ARB_POOL = 0xcDa53B1F66614552F834cEeF361A8D12a0B8DaD8;
+    uint256[] memory pools = new uint256[](1);
+    pools[0] = ONE_FOR_ZERO_MASK + uint256(uint160(USDC_ARB_POOL));
+
+    bytes memory oneInchApiDataUsdcToArb = abi.encode(usdcAmount, minReturn, pools);
```

Comment:
i've already figured out how to encode it without using 1inch api. I though it would be more simple to use 1inch api. Cc: @rhlsthrm

Reply:
^^ https://github.com/connext/connext-integration/pull/7#discussion_r1166190019

Reply:
The idea here is we should be able to use 1inch API and create adapters for any other API or DEX and get a response which we can send directly into this contract. And if we use `directSwap` we don't need to do any re-encoding on the source side.

Reply:
got it 

https://github.com/connext/connext-integration/pull/7/commits/fe674c1b9ac35ab3fdd7810a2434bb104c42f9f8